lib/bot.js

```diff
@@ -390,6 +436,19 @@ class Bot {
       return match;
     });
 
+    // Webhooks first
+    const webhook = this.findWebhook(channel);
+    if (webhook) {
+      logger.debug('Sending message to Discord via webhook', withMentions, channel, '->', `#${discordChannel.name}`);
+      const avatarURL = this.getDiscordAvatar(author, channel);
```

Comment:
During netsplits `author` is nil. This would usually send some message like `<${username}> ${message}` (or something similar, I can't recall. but the message would contain `${}` sent verbatim). Now `this.getDiscordAvatar` crashes with `TypeError: Cannot read property 'toLowerCase' of undefined`